OpenRA.Mods.Common/Traits/TerrainLighting.cs

```diff
@@ -0,0 +1,140 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using OpenRA.Primitives;
+using OpenRA.Support;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	public class TerrainLightingInfo : TraitInfo, ILobbyCustomRulesIgnore
```

Comment:
Maybe explain in a `[Desc("")]` that this belongs on the world actor and what it does.

Reply:
Good idea. Done.